src/devices_models/devices/static_injection_security_constrained_models.jl

```diff
@@ -358,349 +506,611 @@ function add_constraints!(
             T(),
             R,
-            IS.get_uuid.(associated_outages),
-            retained_branches_names,
+            string.(IS.get_uuid.(associated_outages)),
+            device_names,
             time_steps;
-            meta = "$service_name -ub",
-        )
-
-    expressions = get_expression(container, U(), R, service_name)
-
-    #param_keys = get_parameter_keys(container)
-
-    for branch in branches
-        branch_name = PSY.get_name(branch)
-        if !(branch_name in retained_branches_names)
-            @debug "Branch $branch_name not in the reduced network, skipping post-contingency rate limit constraints."
-            continue
-        end
-
-        # param_key = ParameterKey(
-        #     PostContingencyDynamicBranchRatingTimeSeriesParameter,
-        #     typeof(branch),
-        # )
-        # has_dlr_ts = (param_key in param_keys) && PSY.has_time_series(branch)
-
-        #device_dynamic_branch_rating_ts = []
-        # if has_dlr_ts
-        #     device_dynamic_branch_rating_ts, mult =
-        #         _get_device_post_contingency_dynamic_branch_rating_time_series(
-        #             container,
-        #             param_key,
-        #             branch_name,
-        #             network_model)
-        # end
-        limits = get_min_max_limits(
-            branch,
-            PostContingencyEmergencyRateLimitConstrain,
-            AbstractBranchFormulation,
-            network_model,
+            meta = "ub",
         )
-
-        for outage in associated_outages
-            outage_name = IS.get_uuid(outage)
-
-            for t in time_steps
-                # device_dynamic_branch_rating_ts is empty if this device doesn't have a time series
-                # if !isempty(device_dynamic_branch_rating_ts)
-                #     limits = (
-                #         min = -1 * device_dynamic_branch_rating_ts[t] *
-                #               mult[branch_name, t],
-                #         max = device_dynamic_branch_rating_ts[t] * mult[branch_name, t],
-                #     ) #update limits
-                # end
-
-                con_ub[outage_name, branch_name, t] =
-                    JuMP.@constraint(get_jump_model(container),
-                        expressions[outage_name, branch_name, t] <=
-                        limits.max)
-                con_lb[outage_name, branch_name, t] =
-                    JuMP.@constraint(get_jump_model(container),
-                        expressions[outage_name, branch_name, t] >=
-                        limits.min)
+    expressions = get_expression(container, U(), R, "")
+    for ac_type in ac_transmission_types
+        !(ac_type in network_model.modeled_branch_types) && continue
+        for map in NETWORK_REDUCTION_MAPS
+            network_reduction_map = all_branch_maps_by_type[map]
+            !haskey(network_reduction_map, ac_type) && continue
+            #!haskey(network_model.modeled_branch_types, ac_type) && continue
```

Comment:
The check here replacing line 1073 and 1074 should be: `!(ac_type in modeled_branch_types) && continue`. modeled_branch_types is what is used when constructing the container so this will ensure we are matching the container.